include/json/value.h

```diff
@@ -323,7 +323,10 @@ Json::Value obj_value(Json::objectValue); // {}
   int compare(const Value& other) const;
 
   const char* asCString() const; ///< Embedded zeroes could cause you trouble!
-  std::string asString() const; ///< Embedded zeroes are possible.
+#if JSON_USE_SECURE_MEMORY
+  unsigned getCStringLength() const; //Allows you to understand the length of the CString
```

Comment:
Hi Chris, it is saving me in my code from doing 
`auto tmp = myVal.asString(); myNonSecure = std::string(tmp.data(), tmp.data().length()); `
instead i can do
`auto tmp = std::string; myNonSecure = std::string(myVal.asCString(), myVal.getCStringLength());`

I think it's worth having because it saves me doing at least one copy operation.